XamarinCommunityToolkitSample/ViewModels/AboutViewModel.cs

```diff
@@ -46,30 +49,29 @@ public string EmptyViewText
 
         public async Task OnAppearing()
         {
-            if (Contributors != null)
+            if (Contributors.Any())
```

Comment:
I think this check is wrong now. Before we would return when `Contributors != null` now we return whenever the `Contributors` has `Any()`

Reply:
1) That's impossible to have 0 contributors on the GitHub project (only if not you've just created it without readme)
2) From my perspective, if for some reason we pulled 0 contributors on OnAppearing, we should try to pull it again on OnAppearing.

Also now Contributors always has value.